dashboard/test/models/script_test.rb

```diff
@@ -2071,7 +2073,7 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
         ScriptDSL.parse(dsl, 'a filename')[0][:lesson_groups]
       )
     end
-    assert_equal 'Expect if one lesson has a lesson group all lessons have lesson groups. Lesson Lesson1 does not have a lesson group.', raise.message
+    assert_equal 'Expect if one lesson has a lesson group all lessons have lesson groups.', raise.message
```

Comment:
Moved where the check for this is so we don't have the lesson information as easily.